### PR TITLE
Session

### PR DESCRIPTION
**Pull request by divs4debu**

### CODE REVIEW

**Comment by divs4debu** [comment]
@iamareebjamal Please review

**Comment by divs4debu** [comment]
@iamareebjamal  Please review

File: src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java [hunk 1]
```diff
@@ -71,15 +73,15 @@ public String createCourse(@Valid Course course, BindingResult bindingResult, Re
...
+    public String floatCourse(@RequestParam String courseCode, @RequestParam List<String> faculty, RedirectAttributes redirectAttributes, @RequestParam String session) {
         Course course = departmentAdminService.findCourseByCode(courseCode);
 
         List<String> errors = new ArrayList<>();
         if (course == null) {
             errors.add("No valid course selected");
         } else {
             try {
-                departmentAdminService.floatCourse(course, faculty);
+                departmentAdminService.floatCourse(course, faculty).setSession(session);
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java]: We don't want to set session. Only show it

File: src/main/java/in/ac/amu/zhcet/controller/FacultyController.java [hunk 2]
```diff
@@ -49,6 +50,7 @@ public String getStudents(Model model, @PathVariable String id) {
         List<CourseRegistration> courseRegistrations = floatedCourse.getCourseRegistrations();
         model.addAttribute("courseRegistrations", courseRegistrations);
         model.addAttribute("course_id", id);
+        model.addAttribute("currentSession", Utils.getCurrentSession());
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/controller/FacultyController.java]: Thymeleaf template should automatically access the static method. There is no need to include it in every model

File: src/main/java/in/ac/amu/zhcet/data/service/DepartmentAdminService.java [hunk 3]
```diff
@@ -44,13 +44,13 @@ public void registerCourse(Course course) {
         floatedCourseService.register(course);
     }
 
-    public void floatCourse(Course course, List<String> facultyMembers) throws IllegalAccessException {
+    public FloatedCourse floatCourse(Course course, List<String> facultyMembers) throws IllegalAccessException {
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/data/service/DepartmentAdminService.java]: This needs to be void

File: src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java [hunk 4]
```diff
@@ -4,6 +4,7 @@
 import in.ac.amu.zhcet.data.model.FacultyMember;
 import in.ac.amu.zhcet.data.service.DepartmentAdminService;
 import in.ac.amu.zhcet.data.service.FacultyService;
+import in.ac.amu.zhcet.utils.Utils;
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java]: Remove unused imports as Codacy comments

**Pull request closed without merging**